### DISCUSSION TITLE
How do I change the name/icon on the PWA?

### DISCUSSION THREAD

**Discussion by zoeleu** [coder/code-server#4104] [Q&A]
I think they look ugly, how can I change them?

**Comment by lg-nineoneninethreenine** [comment]
Can you tell which OS you are using? It is possible on Windows easily but I don't know about Mac OS and Linux.

**Comment by zoeleu** [comment]
Can I change it on the server?

Em dom., 5 de set. de 2021 às 00:00, LG ***@***.***> escreveu:

**Reply by lg-nineoneninethreenine** [reply]
I don't know.

**Comment by bpmct** [comment]
I believe this is hardcoded in code-server in [./src/browser/media](https://github.com/cdr/code-server/tree/main/src/browser/media) you can either change this manually and rebuild, or create an issue/feature request to make this configurable via the CLI.

**Comment by JoJk0** [comment]
Due to using secure context (HTTPS), PWA cannot be renamed or icon changed by the user. It would be cool to customize it via config

**Reply by JoJk0** [reply]
If in Ubuntu, you can change it without rebuilding here: `\usr\lib\code-server\src\browser\media`

**Reply by JoJk0** [reply]
You can change manifest in `/usr/lib/code-server/out/node/routes/vscode.js` inside there is `manifest.json` factory in where you can change whatever you need :)
